docs/ref/models/database-functions.txt

```diff
@@ -293,3 +293,518 @@ Usage example::
 .. versionchanged:: 1.9
 
     The ability to register the function as a transform was added.
+
+Date Functions
+==============
+
+.. module:: django.db.models.functions.date
+
+.. versionadded:: 1.10
+
+We'll be using the following model in examples of each function::
+
+    class Experiment(models.Model):
+        start_time = models.DateTimeField()
+        end_time = models.DateTimeField(null=True, blank=True)
+
+``Extract``
+-----------
+
+.. class:: Extract(date_expression, lookup_name=None, **extra)
+
+Extracts a component of a date as a number.
+
+Takes a single ``date_expression`` representing a ``DateField`` or
+``DateTimeField`` and a ``lookup_name``, and returns the part of the date
+referenced by ``lookup_name`` as an ``IntegerField``. Django will generally use
+the databases Extract function, so ``lookup_name``\s that aren't explicitly
+referenced below can still be used provided the database supports that value.
+
+Given the datetime `2015-06-15 14:30:50.321` the ``lookup_name``\s supported by
+Django will return:
+
+* "year": 2015
+* "month": 6
+* "day": 15
+* "week_day": From 1 (Sunday) to 7 (Saturday)
+* "hour": 14
+* "minute": 30
+* "second": 50
```

Comment:
Yes, it will, because a given point in time may have a different year, month, day, week_day, hour or minute depending on the timezone it's represented in. Differences in hour or minutes are a direct consequences of time zone offsets; differences in year, month or day happen when you're close to the boundary and the offset makes you cross it.

This is why dates cannot be treated as a a less precise version of datetimes in general.